components/Login/ResetPassword.js

```diff
@@ -56,12 +79,46 @@ class ResetPassword extends React.Component {
     }
 
     componentDidUpdate() {
+        const messages = defineMessages({
```

Comment:
Why do you initialize this static content on each function call? Initializing it in the constructor and storing under this is what I would suggest.

Reply:
Never tested storing the variable in "this.state"...just oriented myself as of the provided examples from confluence (where this is also initialised in functions).

Reply:
Hm OK. No need to change it.